yt/data_objects/tests/test_data_containers.py

```diff
@@ -18,174 +17,179 @@
-
-    def test_std(self):
-        ds = fake_random_ds(3)
-        ds.all_data().std(("gas", "density"), weight=("gas", "velocity_z"))
-
-    def test_to_frb(self):
-        # Test cylindrical geometry
-        fields = ["density", "cell_mass"]
-        units = ["g/cm**3", "g"]
-        ds = fake_amr_ds(
-            fields=fields, units=units, geometry="cylindrical", particles=16**3
-        )
-        dd = ds.all_data()
-        proj = ds.proj(
-            ("gas", "density"),
-            weight_field=("gas", "cell_mass"),
-            axis=1,
-            data_source=dd,
-        )
-        frb = proj.to_frb((1.0, "unitary"), 64)
-        assert_equal(frb.radius, (1.0, "unitary"))
-        assert_equal(frb.buff_size, 64)
-
-    def test_extract_isocontours(self):
-        # Test isocontour properties for AMRGridData
-        fields = ["density", "cell_mass"]
-        units = ["g/cm**3", "g"]
-        ds = fake_amr_ds(fields=fields, units=units, particles=16**3)
-        dd = ds.all_data()
-        q = dd.quantities["WeightedAverageQuantity"]
-        rho = q(("gas", "density"), weight=("gas", "cell_mass"))
-        dd.extract_isocontours(("gas", "density"), rho, "triangles.obj", True)
-        dd.calculate_isocontour_flux(
-            ("gas", "density"),
-            rho,
-            ("index", "x"),
-            ("index", "y"),
-            ("index", "z"),
-            ("index", "dx"),
-        )
-
-        # Test error in case of ParticleData
-        ds = fake_particle_ds()
-        dd = ds.all_data()
-        q = dd.quantities["WeightedAverageQuantity"]
-        rho = q(("all", "particle_velocity_x"), weight=("all", "particle_mass"))
-        with assert_raises(NotImplementedError):
-            dd.extract_isocontours("density", rho, sample_values="x")
-
-    def test_derived_field(self):
-        # Test that derived field on filtered particles do not require
-        # their parent field to be created
-        ds = fake_particle_ds()
-        dd = ds.all_data()
-        dd.set_field_parameter("axis", 0)
-
-        @particle_filter(requires=["particle_mass"], filtered_type="io")
-        def massive(pfilter, data):
-            return data[pfilter.filtered_type, "particle_mass"].to("code_mass") > 0.5
-
-        ds.add_particle_filter("massive")
-
-        def fun(field, data):
-            return data[field.name[0], "particle_mass"]
-
-        # Add the field to the massive particles
-        ds.add_field(
-            ("massive", "test"),
-            function=fun,
-            sampling_type="particle",
-            units="code_mass",
-        )
-
-        expected_size = (dd["io", "particle_mass"].to("code_mass") > 0.5).sum()
-
-        fields_to_test = [f for f in ds.derived_field_list if f[0] == "massive"]
-
-        def test_this(fname):
-            data = dd[fname]
-            assert_equal(data.shape[0], expected_size)
-
-        for fname in fields_to_test:
-            test_this(fname)
+@pytest.fixture(scope="module")
```

Comment:
I don't know that we need a scope here ?
```suggestion
@pytest.fixture
```